x-pack/plugins/ml/public/application/components/scatterplot_matrix/scatterplot_matrix.scss

```diff
@@ -1,4 +1,6 @@
 .mlScatterplotMatrix {
+  overflow-x: scroll;
```

Comment:
Why didn't you go with `auto` here. Testing on my setup `auto` seems to work ok.

Reply:
You're right of course! I missed it because on MacOS scrollbars are hidden by default even with `scroll`.